### ISSUE TITLE
Add support for SKlearn

### ISSUE THREAD

**Issue by evolu8** [allegroai/trains#20]
Hey guys. 

I'd like to use this mostly for SKLearn. Please advise. e.g. gridsearch support, allowing param recording for search iterations.

**Comment by bmartinn** [comment]
[This](https://github.com/allegroai/trains/blob/master/docs/faq.md#can-i-create-a-graph-comparing-hyper-parameters-vs-model-accuracy-) looks like what you are looking for.

**Comment by bmartinn** [comment]
Closing, due to lack of activity.
